include/cantera/zeroD/ReactorNet.h

```diff
@@ -35,6 +35,9 @@ class ReactorNet : public FuncEval
     //! using the current mixture state as the initial condition.
     void setInitialTime(double time);
 
+    //! Get the maximum time step.
+    double maxTimeStep();
```

Comment:
I think it is typical in Cantera for very simple getter methods to implement the logic in the header file.

Reply:
NP